### ISSUE TITLE
Unable to edit long recording names

### ISSUE THREAD

**Issue by beckerr-rzht** [bigbluebutton/greenlight#1776]
Long recording names can not be edited, because of the edit handle disappears behind the preview image.

Cause is a `overflow: hidden` define on the parent `div.edit_hover_class`.

This css fixes this Issue, by adding word break:
```
div.edit_hover_class { word-break: break-all; white-space: normal; }
```

Do you need a pull request for this?

**Comment by beckerr-rzht** [comment]
This is a screenshot illustrating the problem:

![Screenshot](https://user-images.githubusercontent.com/15359213/84250223-0f210180-ab0c-11ea-9e0d-b6eaa7c1df4b.png)

**Comment by farhatahmad** [comment]
Actually, I'll keep this open until v2.7-alpha branch gets merged into master

**Comment by farhatahmad** [comment]
Fixed in 2.7
